Makefile

```diff
@@ -64,22 +64,27 @@ simplify:
 # Collects all dashboards and generates dashboard folder for https://github.com/elastic/beats-dashboards/tree/master/dashboards
 .PHONY: beats-dashboards
 beats-dashboards:
-	mkdir -p build
-	$(foreach var,$(PROJECTS),cp -r $(var)/etc/kibana/ build/dashboards  || exit 1;)
+	mkdir -p build/dashboards
+	$(foreach var,$(BEATS),cp -r $(var)/etc/kibana/ build/dashboards/$(var)  || exit 1;)
 
 # Builds the documents for each beat
 .PHONY: docs
 docs:
 	sh libbeat/scripts/build_docs.sh ${PROJECTS}
 
 .PHONY: package
-package:
+package: beats-dashboards
 	$(MAKE) -C libbeat package-setup
 	$(foreach var,$(BEATS),SNAPSHOT=$(SNAPSHOT) $(MAKE) -C $(var) package || exit 1;)
 
+	# build the dashboards package
+	BUILD_DIR=${shell pwd}/build SNAPSHOT=$(SNAPSHOT) $(MAKE) -C dev-tools/packer package-dashboards ${shell pwd}/build/upload/build_id.txt
```

Comment:
We should make this available in libbeat. The reason for the refactoring of the packaging was to make packaging available in each beat which includes community beats. Community beats should be able to package their dashboards too. All the implementation in this Makefile should only call the Makefile commands in the betas itself and potentially gather some data.